components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -574,25 +572,6 @@ export abstract class AbstractTypeORMWorkspaceDBImpl implements WorkspaceDB {
             .getOne();
     }
 
-    public async getTotalPrebuildUseSeconds(forDays: number): Promise<number | undefined> {
-        const manager = await this.getManager();
-        const res = await manager.query(`
-            SELECT SUM(COALESCE(STR_TO_DATE(wsi.stoppedTime, "%Y-%m-%dT%H:%i:%s.%fZ") - STR_TO_DATE(wsi.startedTime, "%Y-%m-%dT%H:%i:%s.%fZ"))) AS tps FROM d_b_workspace_instance wsi
```

Comment:
If it was just for the query time we could optimize this for sure. But I assume it's a product-decision to remove this constraint and move to sth else based on growth for Enterprise (e.g., "free for up to 10 seats, pay for more").